packages/button/src/Button/Button.spec.tsx

```diff
@@ -61,11 +61,10 @@ describe('packages/button', () => {
       expect(button.title).toBe(title);
     });
 
-    test(`renders the disabled and aria-disabled attributes when disabled is set`, () => {
+    test(`renders aria-disabled attribute when disabled is set`, () => {
       const { button } = renderButton({
         disabled: true,
       });
-      expect((button as HTMLButtonElement).disabled).toBe(true);
```

Comment:
I can understand why we might want to remove `pointer-events` from the CSS, but why do we remove the `disabled` attribute?

Reply:
Oh I see it plays with why we're no longer checking for the anchor type in the Button component. I see. But shouldn't we still be setting this attribute for <button>s?

Reply:
setting aria-disabled instead.

Reply:
Those are only the same semantically though. We should make sure we have parity in the ways they're different: 
![image](https://user-images.githubusercontent.com/17347644/217361597-6b0e5d6f-c680-48dd-8fcf-84213279899d.png)